.env

```diff
@@ -1,3 +1,4 @@
 COMPOSE_PROJECT_NAME=sciencebeam-trainer-grobid
 GROBID_TAG=0.5.4
-IMAGE_TAG=develop
+IMAGE_TAG=0.5.4-develop
```

Comment:
maybe a comment like `# $version-$commit` or similar helps documenting this format